src/pages/IndexPage/components/IndexBus/hooks/useIndexBusDirection.ts

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react';
+
+const useIndexBusDirection = () => {
+  const [toSchoolList, setToSchoolList] = useState([false, false, false]);
+
+  const toggleDirection = (index: number) => {
+    const newList = [...toSchoolList];
+    newList[index] = !newList[index];
+    setToSchoolList(newList);
```

Comment:
```suggestion
    setToSchoolList((value) => {
     const newValue = [...value];
     value[index] = !value[index];
     return newValue;
    });
```
으로 외부 변수 참조를 최소화 할 수 있을 것 같습니다.

Reply:
오 좀 더 목적에 알맞네요!
수정했습니다. [7865767](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/29/commits/7865767013da7f725f7e9e241e3fdb184b192f1b)